### DISCUSSION TITLE
My README.md file's preview is not availble on my repository's URL main page.

### DISCUSSION THREAD

**Discussion by sty8470** [community/community#52620] [Repositories] [answered]
### Select Topic Area

Question

### Body

Hello this is my repo's URL: https://github.com/sty8470?tab=repositories

I am supposed to be able to see the preview of README.md files of the following repository such as "[amazon_product_crawler_to_csv](https://github.com/sty8470/amazon_product_crawler_to_csv)"

But I somehow cannot, so I tried a lot of ways but keep failing. I need your assistance. Thanks.

**Comment by airtower-luna** [comment]
> Hello this is my repo's URL: https://github.com/sty8470?tab=repositories

That's not a repository page, that's the list of your own repositories. You'll only see the repository name and description (if any) there, not the README. The README should show on the repository page, where you go if you click a repository name in the list.

**Reply by Anmol-Baranwal** [reply]
Yep :)

**Reply by sty8470** [reply]
Thanks for your response. But I think I also put descriptions on there to be shown. How do I put descriptions legitimately?

**Reply by airtower-luna** [reply, marked as answer]
If you visit the repository page, you should see a little gear symbol next to the "About" section:

![image](https://user-images.githubusercontent.com/5814919/231502454-f659faef-2347-4b86-ae91-7df9be3a674c.png)

Click it, and you should get a modal window where you can edit the description and a few other bits of information about your repository.

**Reply by Lukasz-Grzegorzewski** [reply]
To have a README overview displayed at the top of your GitHub profile page, you need to create a special repository with the same name as your GitHub username. This repository will act as your profile repository and display its README file on your profile.
Create new, public repository (with README file) with the same name as your GitHub username.

**Comment by g-nesh** [comment]
<picture>
  <source
    srcset="https://github-readme-streak-stats.herokuapp.com/?user=g-nesh&theme=dracula&hide_border=true"
    media="(prefers-color-scheme: dark)"
  />
  <source
    srcset="https://github-readme-streak-stats.herokuapp.com/?user=g-nesh&hide_border=false"
    media="(prefers-color-scheme: light), (prefers-color-scheme: no-preference)"
  />
  <img src="https://github-readme-streak-stats.herokuapp.com/?user=g-nesh&hide_border=true" alt="Ganesh's GitHub streaks" />
</picture>

**Comment by g-nesh** [comment]
<picture>
  <source
    srcset="<?php echo $githubStatsURL; ?>&theme=dracula&hide_border=true"
    media="(prefers-color-scheme: dark)"
  />
  <source
    srcset="<?php echo $githubStatsURL; ?>&hide_border=false"
    media="(prefers-color-scheme: light), (prefers-color-scheme: no-preference)"
  />
  <img src="<?php echo $githubStatsURL; ?>&hide_border=true" alt="Ganesh's GitHub streaks" />
</picture>

**Comment by g-nesh** [comment]
<picture>
  <source
    srcset="https://github-readme-streak-stats.herokuapp.com/?user=g-nesh&theme=dracula&hide_border=true"
    media="(prefers-color-scheme: dark)"
  />
  <source
    srcset="https://github-readme-streak-stats.herokuapp.com/?user=g-nesh&hide_border=false"
    media="(prefers-color-scheme: light), (prefers-color-scheme: no-preference)"
  />
  <img src="https://github-readme-streak-stats.herokuapp.com/?user=g-nesh&hide_border=true" alt="Ganesh's GitHub streaks" />
</picture>

**Comment by g-nesh** [comment]
<picture>
  <source
    srcset="https://github-readme-streak-stats.herokuapp.com/?user=g-nesh&theme=dracula&hide_border=true"
    media="(prefers-color-scheme: dark)"
  />
  <source
    srcset="https://github-readme-streak-stats.herokuapp.com/?user=g-nesh&hide_border=false"
    media="(prefers-color-scheme: light), (prefers-color-scheme: no-preference)"
  />
  <img src="https://github-readme-streak-stats.herokuapp.com/?user=g-nesh&hide_border=true" alt="Ganesh's GitHub streaks" />
</picture>
